src/components/Post/Url.js

```diff
@@ -15,9 +15,9 @@ export default function Url({ url }) {
   );
 }
 const UrlStyle = styled.a`
-  max-width: 503px;
+  /* max-width: 503px; */
```

Comment:
**Question:** was this supposed to be here?

Reply:
No! Already removed
Thanks!